app/views/comments/edit.js.erb

```diff
@@ -6,4 +6,8 @@
   $('[data-behavior~=rich-toolbar]', '#<%= dom_id(@comment) %>').each(function() {
     new EditorToolbar($(this));
   });
+
+  $('[data-behavior~=local-auto-save]', '#<%= dom_id(@comment) %>').each(function() {
+    new LocalAutoSave($(this)[0]);
```

Comment:
```suggestion
    new LocalAutoSave(this);
```

If you don't wrap it with `$()` then you don't need to access it from `[0]`

Reply:
Ahh, missed that out. I'll fix it.